scss/navbar.scss

```diff
@@ -0,0 +1,22 @@
+.nav {
+  align-items: center;
+  background: #FFF;
+  box-shadow: 0px 8px 10px $grey;
+  display: flex;
+  height: $navbar-height;
+  justify-content: space-between;
+  left: 0;
+  padding: 0px 30px;
+  position: fixed;
+  right: 0;
+  top: 0;
```

Comment:
this top, right, left, bottom are unnecesary

Reply:
For remove top, right and left are necessary add width 100% and change section.body margin by padding.